### PR TITLE
#102 Improve coverage of API/CRV

### PR DESCRIPTION
**Pull request by rosiereflo**

### CODE REVIEW

File: dynawo/cmake/TestCoverage.cmake [hunk 1]
```diff
@@ -13,7 +13,7 @@ if (NOT (CMAKE_BUILD_TYPE STREQUAL "Debug" OR CMAKE_BUILD_TYPE STREQUAL "TestCov
 endif()
 
 # Compilation flags
-set(COVERAGE_CXX_FLAGS "-g -O0 -fprofile-arcs -ftest-coverage")
+set(COVERAGE_CXX_FLAGS "-g -O1 -fprofile-arcs -ftest-coverage -fno-omit-frame-pointer -fno-optimize-sibling-calls")
```
Comment (gautierbureau) [inline dynawo/cmake/TestCoverage.cmake]: why this change ?

Comment (rosiereflo) [reply dynawo/cmake/TestCoverage.cmake]: Should improve the issue with the coverage (prevent from creating branches for every possibility of a if)
